### PR TITLE
[semver:patch] - DT-2254: 🔧 Add in redis node orb and default to lowe…

### PR DESCRIPTION
**Pull request by petergphillips**
…st elasticache in use

### CODE REVIEW

File: src/executors/node_redis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+description: >
+  Executor for node.js jobs with redis image
+docker:
+  - image: 'cimg/node:<<parameters.node_tag>>'
+  - image: 'circleci/redis:<<parameters.redis_tag>>'
+working_directory: ~/app
+parameters:
+  node_tag:
+    default: "14.16-browsers"
```
Comment (markreesmoj) [inline src/executors/node_redis.yml:10]: The typescript template project, which could make use of this executor, uses 14.17 here

Comment (petergphillips) [reply src/executors/node_redis.yml:10]: agreed, but wanted to get it initally to match the `node` executor

Comment (andrewrlee) [reply src/executors/node_redis.yml:10]: could bump the hmpps/node executor too.

Comment (petergphillips) [reply src/executors/node_redis.yml:10]: so that if people grab both for their projects and reference both without tags they don't get a version mismatch

Comment (andrewrlee) [reply src/executors/node_redis.yml:10]: it's all annoying as there's a load of different variations too - using postgres, mssql, cypress... etc..

Then if you do start using the default version specified here then its really easy for this to silently mismatch the version specified in the docker image.. why can't we just have nice things? 😄

Comment (petergphillips) [reply src/executors/node_redis.yml:10]: agreed - I'm tempted to remove the default version here as it always gets out of date and a nightmare then to change.  People should then explicitly have to specify it

**Review by markreesmoj — APPROVED** [review]
👍

**Pull request merged**